seatunnel-connectors-v2/connector-cdc/connector-cdc-mysql/src/main/java/org/apache/seatunnel/connectors/seatunnel/cdc/mysql/source/parser/CustomMySqlAntlrDdlParser.java

```diff
@@ -0,0 +1,293 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.cdc.mysql.source.parser;
+
+import org.apache.seatunnel.api.table.event.AlterTableColumnEvent;
+
+import com.google.common.collect.Lists;
+import io.debezium.antlr.AntlrDdlParserListener;
+import io.debezium.antlr.DataTypeResolver;
+import io.debezium.connector.mysql.antlr.MySqlAntlrDdlParser;
+import io.debezium.ddl.parser.mysql.generated.MySqlParser;
+import io.debezium.relational.RelationalDatabaseConnectorConfig;
+
+import java.sql.Types;
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+/** A ddl parser that will use custom listener. */
+public class CustomMySqlAntlrDdlParser extends MySqlAntlrDdlParser {
```

Comment:
Override `parseQualifiedTableId` method rewrite database/schema field

Reply:
Good idea. I'll do it.